.github/workflows/build.yml

```diff
@@ -162,17 +109,38 @@ jobs:
           retry_wait_seconds: 10
           command: docker pull docker.pkg.github.com/apache/incubator-nuttx-testing/nuttx-ci-linux
 
-      - name: Export NuttX Repo SHA
-        run:  echo "::set-env name=nuttx_sha::`git -C sources/nuttx rev-parse HEAD`"
-      - name: Refresh Git Credentials
+      - uses: actions/download-artifact@v2
+        with:
+          name: gitrefs
+          path: gitrefs-rt/
+      - name: Export Repo Refs
```

Comment:
move before "uses: actions/download-artifact@v2"

Reply:
All steps have a name but "uses: actions/download-artifact@v2" don't have. so I suppose that it's one step from line 118 to line 123, and should order like this:
      - name: Export Repo Refs
      - uses: actions/download-artifact@v2
        with:
          name: gitrefs
          path: gitrefs-rt/
        run: source gitrefs-rt/gitenv.sh

Reply:
Ah I see the confusion. That is actually two steps. Every '-' is a step, you cannot have a 'run' and a 'uses' in the same step.